test/reads_len.c

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Please remove test/reads_len binary file above. Put test/read_len to .gitignore. the comment is in the wrong place here because I cannot comment on the binary file.